lib/git_reflow/git_server/pull_request.rb

```diff
@@ -156,54 +158,56 @@ def commit_message_for_merge
           message << "\nLGTM given by: @#{lgtm_authors.join(', @')}\n"
         end
 
-        message
+        "#{message}\n"
       end
 
       def cleanup_feature_branch?
-        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to cleanup your feature branch? ") =~ /^y/i
+        GitReflow::Config.get('reflow.always-cleanup') == "true" || (ask "Would you like to push this branch to your remote repo and cleanup your feature branch? ") =~ /^y/i
       end
```

Comment:
this message is specific to Bitbucket and `deliver -f` fallbacks, as in these cases we don't push to remote unless the user requests it.